### PR TITLE
Fixed a number of broken back links

### PR DESCRIPTION
**Pull request by danwilliams**
Fixed a number of broken back links.

### CODE REVIEW

**Review by sanisc — APPROVED** [review]

File: vulnerable_people_form/form_pages/cookies.py [hunk 1]
```diff
@@ -5,4 +5,4 @@
 
 @form.route("/cookies", methods=["GET"])
 def get_cookies():
-    return render_template_with_title("cookies.html", back_url=dynamic_back_url())
+    return render_template_with_title("cookies.html", previous_url=dynamic_back_url())
```
Comment (tal-nagra) [inline vulnerable_people_form/form_pages/cookies.py:8]: The `cookies.html` template appears to refer to back_url
`<a class="govuk-link cookie-settings__prev-page" href={{back_url}}>Go back to the page you were looking at</a>`

Take a look and see if anything needs amending here...

Comment (sanisc) [reply vulnerable_people_form/form_pages/cookies.py:8]: @danwilliams ?

File: vulnerable_people_form/form_pages/not_eligible_england.py [hunk 2]
```diff
@@ -6,5 +6,5 @@
 @form.route("/not-eligible-england", methods=["GET"])
 def get_not_eligible_england():
     return render_template_with_title(
-        "not-eligible-england.html", back_url=dynamic_back_url()
+        "not-eligible-england.html", previous_path=dynamic_back_url()
```
Comment (sanisc) [inline vulnerable_people_form/form_pages/not_eligible_england.py:9]: does this fix this https://trello.com/c/u07YqQJo/280-sorry-youre-not-eligible-for-the-service-update-back-link?

Comment (danwilliams) [reply vulnerable_people_form/form_pages/not_eligible_england.py:9]: Yes 🙂

File: vulnerable_people_form/form_pages/cookies.py (hunk 1 above)
Comment (danwilliams) [reply vulnerable_people_form/form_pages/cookies.py]: Good spot! Corrected 🙂

**Review by sanisc — APPROVED** [review]

**Pull request merged**